src/index.html

```diff
@@ -5,12 +5,13 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Page title</title>
     <link rel="stylesheet" href="./sass/index.scss" />
+  
     <script type="module" src="./index.js"></script>
   </head>
   <body>
     <!-- <h1>Example title</h1> -->
     <!-- ❗️❗️❗️ Add partials like so for every page -->
     <!-- <include src="./partials/example.html"></include> -->
-    <include src="./partials/header.html"></include>
+    <include src="./partials/header.html"></include>  
```

Comment:
Trebuie sa ai mai multe tag-uri de include, ti le-am pus in chat pe slack pe toate